src/server.h

```diff
@@ -1002,6 +1043,13 @@ typedef struct replBufBlock {
     char buf[];
 } replBufBlock;
 
+/* Link list block, used by replDataBuf during rdb-connection sync to store
+ * replication data */
+typedef struct replDataBufBlock {
+    size_t size, used;
+    char buf[];
+} replDataBufBlock;
+
```

Comment:
does this struct need to be used outside of replication.c? if not, let's minimize its scope and move it to replications.c

Reply:
It is only used at replication.c but I'm not sure where else can we define it..

Reply:
replication.c would work for me. I am in favor of the smallest scope.